crawl-ref/source/dat/des/arrival/large.des

```diff
@@ -1110,3 +1110,51 @@ cxx..xxc
 ccG..Gcc
  cc++cc
 ENDMAP
+
+##############################################################################
+NAME:    heliophobic_arrival_dark_side_of_the_moon
+TAGS:    arrival no_monster_gen no_trap_gen
+ORIENT:  float
+SHUFFLE: cx
+MARKER:  F = lua:fog_machine{ \
+             pow_max = 20, delay_min = 20, delay_max = 200, \
+             size = 7, start_clouds = 5, cloud_type = "black smoke"}
+MAP
+        ccccccccc
+      ccccccGcccccc
+     ccc.........ccc
+    cc.....F.......cc
+   cc...............cc
+  cc........cccc.....cc
+  cc......cccccccc...cc
+ cc......cccccccccc...cc
+ cc.....cccccccccccc..cc
+ cc.....cccccccccccc..cc
+ cc....ccccccc...cccc.cc
+ cG{...cccccc..>..ccc..+
```

Comment:
What is the closet with the single hatch for? To be interesting and not just an aesthetic teleport closet, this should be accessible, say with a tunnel
```suggestion
 cG{...cccccc..>.......+
```

Reply:
I thought the teleport closet was a fun little Easter egg. But it was just a little idea. Removed.